src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -219,11 +228,11 @@ private function convertToBytes(string $memoryLimit): int
 
         switch (substr(rtrim($memoryLimit, 'b'), -1)) {
             case 't': $max *= 1024;
-            // no break
+                // no break
             case 'g': $max *= 1024;
-            // no break
+                // no break
             case 'm': $max *= 1024;
-            // no break
+                // no break
             case 'k': $max *= 1024;
```

Comment:
Unrelated change, should be reverted.

Reply:
Fabbot complained about it. Should I ignore it?

Reply:
Yes. You didn't break it, you don't need to fix it.